widget/embedded/src/index.ts

```diff
@@ -6,6 +6,7 @@ import type {
   WidgetColors,
   WidgetColorsKeys,
   WidgetConfig,
+  WidgetEvents,
```

Comment:
You should export enums only as values, not as types.

Reply:
Fixed